tests/neg-macros/delegate-match-2/Macro_1.scala

```diff
@@ -5,7 +5,7 @@ inline def f: Any = ${ fImpl }
 
 private def fImpl (using qctx: QuoteContext) : Expr[Unit] = {
   import qctx.tasty._
-  searchImplicit(('[A]).unseal.tpe) match {
+  ImplicitSearch(('[A]).unseal.tpe) match {
```

Comment:
Minor: usually the convention for `.apply` methods is to simply construct an object from arguments. It seems here `Implicits.search` might be more idiomatic.

Reply:
Indeed. I will try it out